xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/java/org/xwiki/officeimporter/server/OfficeServerConfiguration.java

```diff
@@ -91,6 +101,16 @@ default int[] getServerPorts()
      */
     String getProfilePath();
 
+    /**
+     * @return the path where the files are exchanged between XWiki and the office server (null means: use the default
+     * environment temporary directory)
+     * @see 16.10.0RC1
+     */
+    default String getWorkDir()
+    {
+        return null;
```

Comment:
Not ideal, but I did not want to break the stable interface. The failover is implemented by the component that call `getWorkDir()`.

Reply:
You might want to use `Optional<String>` as the return value, see https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HOptional